src/components/TextInput/TextInput.stories.mdx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import TextInput from "./TextInput";
+import { TextInputTypes } from "./TextInputTypes";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+// eslint-disable-next-line id-blacklist
+import { text, boolean, select, number } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
+
+export default {
+  title: "Form Elements/Text Input",
+  component: TextInput,
+};
+
+<Meta
+  title="Form Elements/Text Input"
+  component={TextInput}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=11895%3A547",
+    },
+  }}
+/>
+
+# TextInput
+
+<Description of={TextInput} />
+
+export const input = () => (
```

Comment:
Using the new MDX format for your stories should allow the user to be able to modify the properties in real-time using the args table.  Recommend having a quick look at Radio component for comparison.

Reply:
@kaplanmichael 
I will start working on converting the `.tsx` to `.mdx`.